ci-operator/templates/openshift/installer/cluster-launch-installer-ipi-e2e.yaml

```diff
@@ -0,0 +1,263 @@
+kind: Template
```

Comment:
Naming here is wrong.  This really upi for packet, so update the naming to be consistent (cluster-launch-installer-upi-packet-e2e).

What's more concerning is that this template is inconsistent in structure, naming, and behavior from the other templates.  This template really needs to look like cluster-launch-installer-e2e with a few small tweaks (such as your custom terraform setup) or be part of that template with a cluster type (packet).  You can omit some of that to get started, but the end result needs to be more consistent with the rest or this will forever atrophy when someone needs to refactor all of them and yours is unmaintainable.

Why is the libvirt image needed?

Reply:
> Naming here is wrong. This really upi for packet, so update the naming to be consistent (cluster-launch-installer-upi-packet-e2e).
> 
> What's more concerning is that this template is inconsistent in structure, naming, and behavior from the other templates. This template really needs to look like cluster-launch-installer-e2e with a few small tweaks (such as your custom terraform setup) or be part of that template with a cluster type (packet). You can omit some of that to get started, but the end result needs to be more consistent with the rest or this will forever atrophy when someone needs to refactor all of them and yours is unmaintainable.

I'm not sure I understand this, we're not using UPI, this is IPI, I don't see it easily fitting into any of the other templates as the flow is different, for the moment we just want to spin up a baremetal centos server and run the installer on it. But I can look at adding the cluster type to cluster-launch-installer-e2e

> 
> Why is the libvirt image needed?

I'm using it because it was the only image I found that had the libnss_wrapper so I could ssh to packet.net servers, incidentally the reason for using the UPI image is because it had terraform.